components/mpas-ocean/src/mode_init/mpas_ocn_init_mode.F

```diff
@@ -267,9 +267,15 @@ function ocn_init_mode_run(domain) result(iErr)!{{{
       type (mpas_pool_type), pointer :: forcingPool
 
       type (MPAS_timeInterval_type) :: timeStep
+      real (kind=RKIND), dimension(:,:), pointer :: normalVelocity
 
       ierr = 0
 
+      ! Initialize normalVelocity to zero for cases that do not specify it.
+      call mpas_pool_get_subpool(domain % blocklist % structs, 'state', statePool)
+      call mpas_pool_get_array(statePool, 'normalVelocity', normalVelocity, 1)
+      normalVelocity(:,:) = 0.0_RKIND
```

Comment:
Note that I did not use a block loop here. Block loops are no longer supported in forward mode, and we never used multiple blocks for init mode anyway, so it seemed silly to add it. We could remove block loops from the rest of the code sometime, but for this PR, I just get the array from the first block. @philipwjones do you agree?

Reply:
I doubt @philipwjones cares if we break init mode ;-)

Reply:
I care deeply and passionately ;^).  But yeah, this is fine - I've been removing them as I find them (block loops that is, not velocity)